From 48635a0fdb06b8572809dc54c630109db1e6e85c Mon Sep 17 00:00:00 2001
From: Matt Traudt <sirmatt@ksu.edu>
Date: Mon, 20 Nov 2017 15:22:51 -0500
Subject: [PATCH] Add space to fmt str in log_gz_file_write

Without this space, the log level identifier ('D', 'I', etc.)
would be right next to actual content.

Before: <timestamp> Depoll->pcb(17): Open event.
After: <timestamp> D epoll->pcb(17): Open event.
---
 src/torrent/utils/log.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/torrent/utils/log.cc b/src/torrent/utils/log.cc
index 646d36e3..58b563a6 100644
--- a/src/torrent/utils/log.cc
+++ b/src/torrent/utils/log.cc
@@ -381,7 +381,7 @@ log_gz_file_write(std::shared_ptr<log_gz_output>& outfile, const char* data, siz
 
   // Normal groups are nul-terminated strings.
   if (group >= 0) {
-    const char* fmt = (group >= LOG_NON_CASCADING) ? ("%" PRIi32 " ") : ("%" PRIi32 " %c");
+    const char* fmt = (group >= LOG_NON_CASCADING) ? ("%" PRIi32 " ") : ("%" PRIi32 " %c ");
 
     int buffer_length = snprintf(buffer, 64, fmt,
                                  cachedTime.seconds(),
